docs/source/conf.py

```diff
@@ -1898,6 +1898,7 @@
     "check_compiler_ok_for_platform",
     "get_cxx_compiler",
     "get_default_build_root",
+    "remove_default_build_root",
```

Comment:
Please make sure to document newly added APIs by adding the appropriate entry in the .rst file+docstring or make them private.